Polyfrost/Hytils-Reborn
# new: hide hud

## Description
Hides certain HUD elements in servers where they don’t change. This is a very old accepted Hytilities suggestion that was never realized due to its discontinuation and it always made me sad.

Things that can be hidden:
- Health
- Food
- Armor
- Air

Might have work to do still (such as reverting the oneconfig gradle version) but PRing for feedback or something idk

## Related Issue(s)

N/A

## How to test
Join a game or lobby where you wouldn’t expect to see a certain HUD element. You should now see that it is hidden. The lobby is a good place to see all four in action. Similarly, for housing and lobby, if you were to take damage it will reintroduce the health bar.

## Release Notes

```release-note
new: hide hud feature to hide hud elements in game modes where they wouldn’t change
```

## Documentation
